src/Common/AbstractModel.m

```diff
@@ -143,6 +145,265 @@ function saveObj(obj, suffix)
             end
 
         end
+
+        function idx = getButtonIdx(obj,buttonName)
+        % This function returns obj.buttons index of the buttonName.
+        % All prepended jokers must be included to this list.
```

Comment:
Can you define explicetely here or somewhere else what you mean by joker again – it's a technical jargon, and although you told me what it means I already forgot, and so that doesn't bode well for  the future developper who will read this when we are both retired in 40 years.

Reply:
I've added some of the functions (the short ones those input/output volume) as a subfunction to the classdef to save some more memory. This was one of them.